module/support/SignalCatcher/src/SignalCatcher.cpp

```diff
@@ -24,12 +24,14 @@
 namespace module::support {
 
     // Set our initial shutdown request state
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
     volatile bool userRequestedShutdown = false;
 
     // Initialize our powerplant variable
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
     NUClear::PowerPlant* POWER_PLANT = nullptr;
 
-    void sigint(int) {
+    void sigint(int /*unused*/) {
```

Comment:
instead of `/*unused*/` in these, we should put /*signal*/ to show what the variable actually would be named if we used it